appveyor.yml

```diff
@@ -1,5 +1,9 @@
 os: Visual Studio 2019
 
+init:
+ - netsh int ipv4 set dynamicport tcp start=1025 num=64510
+ - powershell "New-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Services\Tcpip\Parameters' -Name 'TcpTimedWaitDelay' -Value '15' -PropertyType 'DWord'"
```

Comment:
This setting requires this other key to be set or it will have no effect: ([Source](https://support.solarwinds.com/SuccessCenter/s/article/Tweaking-Windows-Server-performance-to-prevent-port-exhaustion-when-using-Orion-modules?language=en_US#:~:text=NOTE%3A%C2%A0You%C2%A0must%C2%A0set%20the%20StrictTimeWaitSeqCheck%20to%201%2C%20or%20the%20TcpTimedWaitDelay%20value%20will%20have%20no%20effect%20on%20the%20system.))

[HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\Tcpip\Parameters]
"StrictTimeWaitSeqCheck"=dword:00000001

It also only takes effect after reboot, so if the AppVeyor VM reverts to base settings on each reboot this will also have no effect.

Reply:
Yes, AppVeyor build VM is started fresh from image. So there is no point in setting this registry setting... do you know of any other method for setting wait delay that would be effective immediately?

Reply:
Perhaps we can set the per-socket timeout when running the test Suite?

```
// sets the amount of time to linger after closing, using the LingerOption public property.
LingerOption lingerOption = new LingerOption (true, 10);
```

Reply:
Cool, this would work. But I think this amount of changes is beyond this PR and should be discussed further. For now I'll just  remove the wait delay change as is is ineffective and leave the enlargement of client port pool in this PR.